main.go

```diff
@@ -275,9 +291,9 @@ func main() {
 		// Setup metrics endpoint at another server
 		prom.SetMetricsPath(echoPrometheus)
 		go func() {
-			echoPrometheus.Logger = logger
-			echoPrometheus.Logger.Infof("Starting prometheus on port %d", svc.Config.PrometheusPort)
-			echoPrometheus.Logger.Fatal(echoPrometheus.Start(fmt.Sprintf(":%d", svc.Config.PrometheusPort)))
+			//echoPrometheus.Logger = logger
```

Comment:
need to review and update the usage of logger here? can the zerolog logger be used?

Reply:
The zerolog logger can certainly be used here.